### ISSUE TITLE
About learning rate

### ISSUE THREAD

**Issue by LZHgrla** [tinyvision/DAMO-YOLO#103]
### Before Asking

- [X] I have read the [README](https://github.com/tinyvision/DAMO-YOLO/blob/master/README.md) carefully. 我已经仔细阅读了README上的操作指引。

- [ ] I want to train my custom dataset, and I have read the [tutorials for finetune on your data](https://github.com/tinyvision/DAMO-YOLO/blob/master/README.md#Finetune%20on%20your%20data) carefully and organize my dataset correctly; 我想训练自定义数据集，我已经仔细阅读了训练自定义数据的教程，以及按照正确的目录结构存放数据集。

- [X] I have pulled the latest code of main branch to run again and the problem still existed. 我已经拉取了主分支上最新的代码，重新运行之后，问题仍不能解决。

### Search before asking

- [X] I have searched the DAMO-YOLO [issues](https://github.com/tinyvision/DAMO-YOLO/issues) and found no similar questions.

### Question

The learning rate reported on your paper is 0.4, but it is 0.01/64 * 256 ([link](https://github.com/tinyvision/DAMO-YOLO/blob/49dd0dbdcbf9ec67cb9c7b1ae2b571f71449b9db/configs/damoyolo_tinynasL20_T.py#L18)) and 0.001/64 * 256 ([link](https://github.com/tinyvision/DAMO-YOLO/blob/49dd0dbdcbf9ec67cb9c7b1ae2b571f71449b9db/configs/damoyolo_tinynasL20_N.py#L18)) in your code.

I'm wonder that which one is correct? Thanks!
